server/model/Organizations.js

```diff
@@ -1,6 +1,11 @@
 const db = require('./db');
 
 module.exports = {
+  
+  async getAll(id, cb) {
+    return db.query(`select * from organization`, cb);// where id = ${id}`, cb)
+  },
```

Comment:
What does this function do that getOrg below doesn't do with greater specificity?